gatsby-node.js

```diff
@@ -227,12 +235,20 @@ exports.onCreatePage = ({ page, actions }) => {
   // Recreates the page with the messages that ReactIntl needs
   // This will be passed to the ReactIntlProvider Component
   // Used within gatsby-browser.js and gatsby-ssr.js
+  const context = { ...page.context };
+  const locale = context.locale || nodeLocales.defaultLanguage;
+  if (context.isLearnPage) {
```

Comment:
You don't need to do this, do a console.log on the `page` it will tell you the type of the node there should be enough info to tell you if it is a learn page 🤔 

Remember, the `context` object goes to React, so you want to avoid adding stuff there.

Reply:
From context I can see categoryName as learn shall I use that? @ovflowd

Reply:
I've updated the code and removed the isLearnPage from context @ovflowd